src/bidiTab/mapperTabPage.ts

```diff
@@ -36,16 +36,16 @@ function findOrCreateTypeLogContainer(logType: string): HTMLElement {
     return existingContainer;
   }
 
-  const debugElement = document.getElementById('details')!;
+  const debugElement = document.getElementById('details');
```

Comment:
I would highly suggest testing if empty and throwing an error as it may become harder find the issue in the feature (especially for headless).